05/app.js

```diff
@@ -0,0 +1,48 @@
+import Validation from "./validation.js";
+import { getAverageGrade } from "./utils.js";
```

Comment:
Jeśli jest to "utils" czyli uniwersalne narzędzie to czy faktycznie warto dawać tutaj przyrostek `Grade`? To średnia arytmetyczna dla wszystkiego nie tylko dla ocen. W mojej ocenie warto podejść bardziej elastycznie